.gitignore

```diff
@@ -91,6 +91,7 @@ logs/
 mav.log
 mav.log.raw
 mav.parm
+defaults.parm
```

Comment:
this would ignore valid files

Reply:
this is only for the root of ardupilot, it won't impact file in like AP_HAL_CHIBIOS/hwdef